openmc/universe.py

```diff
@@ -219,6 +219,12 @@ def __repr__(self):
         string += '{: <16}=\t{}\n'.format('\tCells', list(self._cells.keys()))
         return string
 
+    def __contains__(self, cell):
+        if self._cells is None:
+            return False
+        else:
+            return cell.id in list(self._cells.keys())
```

Comment:
Why are you checking `cell.id` and not just `cell`? Also `list` is completely unnecessary, and will make this operation 2*O(N), instead of O(1).